src/WalletProviderTester.ts

```diff
@@ -49,6 +49,18 @@ class WalletProviderTester {
     //
     // createSignedTx
     //
+    public async testCreateSignedTx(): Promise<boolean> {
+        const txid = await this.walletProvider.createSignedTx([
+            {
+                lockScript: "76a91467b2e55ada06c869547e93288a4cf7377211f1f088ac",
+                amount: 10000
+            }
+        ])
+        if (typeof txid !== "string") {
+            throw new Error("The return value is invalid.")
+          }
```

Comment:
return value should be tx instead of txid.

Reply:
https://web3bch.github.io/bch-wallet-bridge-provider-interface/interfaces/_iwalletprovider_.iwalletprovider.html#createsignedtx

>The signed raw transaction in serialized transaction format encoded as hex. The provider should throw an error when the transaction is not generated.